src/site/layouts/home.drupal.liquid

```diff
@@ -0,0 +1,139 @@
+{% include "src/site/includes/header.html" %}
+<main>
+
+  <!-- the hub -->
+  <section class="homepage-hub">
+
+    <div class="homepage-hub-container">
+
+      <h1 class="heading-level-2 homepage-heading">Access and manage your VA benefits and health care</h1>
```

Comment:
Looks like the content of this `<h1>` is `{{ heading }}` in `layouts/home.html` (where I think this is mostly copied from?). What's the reasoning for not making it dynamic now?

Reply:
Makes sense. :+1: